### ISSUE TITLE
Usage statistics - differentiating between the test plans and real plans

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2158]
Currently the usage statistics dashboard does not differentiate between the test plans and the real plans. Could we differentiate these for the administrators? 

Let me know if you need more detail about this.

**Comment by briri** [comment]
Yes. It is easy for us to determine which of the plans are 'test'. Do we prefer to exclude them entirely from the statistics calculations or just indicate how many of the total number of plans are 'test'?

**Comment by magdalenadrafiova** [comment]
If we could specify which are the test and which are total numbers that would be fab 🙂


Magdalena Drafiova

Customer Development Manager

Digital Curation Centre<http://www.dcc.ac.uk/>, DMPonline

e. magdalena.drafiova@ed.ac.uk<mailto:magdalena.drafiova@ed.ac.uk>
p. +44 (0)141 330 6749
t. @<https://twitter.com/mdDCC1>mdDCC1<https://twitter.com/mdDCC1><https://twitter.com/urbanbigdata>

 http://www.dcc.ac.uk/events/idcc20



________________________________

**Comment by sjDCC** [comment]
We don't need lots of detail on test plans so exclude from total statistics. If we include anything on test plans just a total number is sufficient

**Comment by sjDCC** [comment]
Make it a toggle so people can see both stats

**Comment by mariapraetzellis** [comment]
Add a checkbox which is defaulted to ticked to include test plans in stats. 
Allow org admins to untick the box
Add text to show total plan figure to show this includes test as shown in screenshot below:

![screenshot-dmptool org-2020 01 09-14_14_43](https://user-images.githubusercontent.com/27511575/72111091-a7f9e980-32ee-11ea-8c8f-4bdf7f2bfb44.png)

**Comment by sjDCC** [comment]
Suggestion from @xsrust that we may need two stats tables - one with test and one without

**Comment by xsrust** [comment]
If it's possible to have a number of test plans in each statistic so we can re-compute the totals with/without tests rather than double the entries in the stats table

**Comment by sjDCC** [comment]
Have validated with @mariapraetzellis and this approach is fine
